Source/WebCore/platform/SharedBuffer.cpp

```diff
@@ -67,6 +69,33 @@ Ref<FragmentedSharedBuffer> FragmentedSharedBuffer::create(DataSegment::Provider
     return adoptRef(*new FragmentedSharedBuffer(WTFMove(provider)));
 }
 
+#if PLATFORM(COCOA)
+std::optional<Ref<FragmentedSharedBuffer>> FragmentedSharedBuffer::fromIPCData(FragmentedSharedBuffer::FragmentedSharedBufferData&& fragmentedSharedBufferData)
```

Comment:
`FragmentedSharedBuffer::` is unnecessary.

Reply:
it wasn't finding it without `FragmentedSharedBuffer::`

Reply:
I am talking about the parameter type -> `FragmentedSharedBuffer::FragmentedSharedBufferData&&` should be `FragmentedSharedBufferData&&`

Reply:
I know, it wasn't finding it. It seems to be working now with IPCData though